src/main/java/command/Command.java

```diff
@@ -0,0 +1,111 @@
+package command;
+
+import task.Task;
+
+/**
+ * Represents a command that the user inputs.
+ */
+public class Command {
+    public enum CommandType {
+        List, Add, Mark, Unmark, Delete, Find, Exit
```

Comment:
Is it better to be consistent in the way you name the enum's values? You are using CamelCase for ``CommandType`` and SCREAMING_SNAKE for ``Task.TYPE``